context-py/src/lib.rs

```diff
@@ -96,6 +97,14 @@ pub fn parse_meta_from_tarball(
     Ok(BindingsVersionedBundle(versioned_bundle))
 }
 
+#[gen_stub_pyfunction]
+#[pyfunction]
+fn codeowners_parse(codeowners_bytes: Vec<u8>) -> PyResult<CodeOwners> {
+    let codeowners = CodeOwners::parse(codeowners_bytes);
+
+    Ok(codeowners)
+}
```

Comment:
No need for `PyResult` since nothing can fail here

```suggestion
fn codeowners_parse(codeowners_bytes: Vec<u8>) -> CodeOwners {
    CodeOwners::parse(codeowners_bytes)
}
```